packages/babel-preset-react-app/create.js

```diff
@@ -45,14 +45,22 @@ module.exports = function(api, opts, env) {
         require('@babel/preset-env').default,
         {
           targets: {
-            node: '6.12',
+            node: 'current',
```

Comment:
This somehow got goofed during an old merge commit, but `current` is the behavior we want.